smartapp/smartapplication/templates/index.html

```diff
@@ -26,9 +26,48 @@
 					<p> <label class="switch"> {{ form.remove_stop_words }}
 						<span class = "slider round"> </span> </label> Remove Stop Words of Your Paragraph</p>
 				</div>
+			<a href="guide">
+				<button disabled class="submit">Submit</button>
+			</a>
 		</div>
-		<a href="guide">
-			<button class="submit">Submit</button>
-		</a>
 	</form>
+<script>
```

Comment:
you might probably consider moving this to a separate file.

Reply:
Created a separate file.